src/strategies/index.ts

```diff
@@ -201,6 +203,8 @@ export default {
   brightid,
   'inverse-xinv': inverseXINV,
   modefi,
+  'modefiStaking': modefiStaking,
```

Comment:
The strategy name should be like this.. can u plz rename the folder name too.. 

```suggestion
  'modefi-staking': modefiStaking,
```